docs/design/plugin_multi_fdb_support.md

```diff
@@ -0,0 +1,134 @@
+# Multi FDB cluster support in the kubectl FDB plugin
+
+## Metadata
+
+* Authors: @johscheuer
+* Created: 2021-03-06
+* Updated: 2021-03-06
+
+## Background
+
+The FDB Kubernetes operator supports to setup and operate FDB clusters span across
+multiple Kubernetes clusters. Currently the setup is more or less manual with the
+following steps: bootstrap the seed cluster and once the seed cluster is bootstrapped
+( actually reconciled) we can adjust the database configuration and bootstrap additional
+FDB clusters managed by the Operator. Updating the cluster means adjusting all manifests
+across the Kubernetes clusters at the same time (or in a short time span). Additionally
+the tooling for managing FDB clusters (the kubectl FDB plugin) is currently not aware of
+FDB clusters spread across multiple Kubernetes clusters or even multiple manifests.
+
+## General Design Goals
+
+* Propose a way how to define where a cluster is located.
+* Bootstrap HA FDB clusters with `kubectl fdb`.
+* Make `kubectl fdb` aware of multiple FDB clusters forming an HA cluster.
+
+## Proposed Design
+
+The idea is to use two additional fields in the CRD that will be ignored by the operator but used by the `kubectl fdb plugin`.
+The fields are `seedCluster` and `targetClusters`, both contain the following `cluster objects`:
+
+```yaml
+name: sample-cluster-dc3 # Optional: Will overwrite the cluster name in the resulting manifest otherwise uses the metadata
+dcID: dc3 # Must match with the data center ID specified under datacenters
+context: my-kube-ctx 3 # Optional: the kubconfig context to use otherwise the current context will be used
+namespace: dc3 # Optional: Otherwise the namespace from the metadata will be used
+processCounts: # Optional: otherwise the processCounts from he cluster Spec will be used (useful to change cluster specific fields e.g. satellites without SS)
```

Comment:
I think this is valid, but the operator should be able to infer the satellite process counts based on matching the DC ID with the region configuration.

Reply:
That's right, we can also remove this part in the first version.